cmd/rollkit/main.go

```diff
@@ -8,9 +8,23 @@ import (
 	"github.com/cometbft/cometbft/libs/cli"
 
 	cmd "github.com/rollkit/rollkit/cmd/rollkit/commands"
+	rollconf "github.com/rollkit/rollkit/config"
 )
 
 func main() {
+	// In case there is a rollkit.toml file in the current dir or somewhere up the
+	// directory tree - we want to intercept the command and execute it against an entrypoint
+	// specified in the rollkit.toml file. In case of missing toml file or missing entrypoint key
+	// or missing actual entrypoint file - the normal rootCmd command is executed.
+	if err := cmd.InterceptCommand(
+		rollconf.ReadToml,
+		cmd.RunRollupEntrypoint,
+	); err == nil {
+		return
+	} else if err != nil {
+		fmt.Fprintf(os.Stderr, "rollkit: %v\n", err)
+	}
```

Comment:
small nit but you don't need this second if condition since err is either nil or not nil. 
I also wouldn't send it to Stderr since running rollkit start in an empty directory with the intention of using rollkit start it initialize the quick start is a normal and expected use case. 

Removing it allows us to then print a more helpful log statement like this
```suggestion
	}
	
	fmt.Fprintf(os.Stdout, "No rollkit.toml file found: %v\nStarting fresh rollup in .rollkit directory\n", err)
```

Reply:
Good point